package.json

```
@@ -3,12 +3,14 @@
   "version": "1.0.0",
   "description": "Mercurius Auth Plugin adds configurable Authentication and Authorization support to Mercurius.",
   "main": "index.js",
+  "types": "index.d.ts",
   "scripts": {
     "unit": "tap --100 test/*.js",
     "cov": "tap --coverage-report=html -J test/*.js",
     "lint": "npm run lint:standard",
-    "lint:standard": "standard | snazzy",
-    "test": "npm run lint && npm run unit"
+    "lint:standard": "ts-standardx --fix | snazzy",
```

No need to install `ts-standardx`, please keep with `standard` and use the options to support typescript. You will need to install:

```json
"@typescript-eslint/eslint-plugin": "^3.6.1",
"@typescript-eslint/parser": "^3.6.1",
```

```suggestion
    "lint:standard": "standard | snazzy",
    "lint:typescript": "standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin test/types/*.ts",
```

---

this one is not working